.github/actions/test-khiops-on-iris/action.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Test Khiops on Iris
+description: Tests a Khiops installation on the Iris dataset
+runs:
+  using: composite
+  steps:
+    # Python install: we don't use the setup-python action because of the following error:
+    # python3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by python3)
+    - name: Python setup for Linux
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        if [ -d "/etc/apt" ]
+        then
+          apt-get install -y python3 > /dev/null
+        else
+          yum install -y python3.11
+        fi
+        echo "PYTHON=python3" >> "$GITHUB_ENV"
+    - name: Python setup for Windows or macOS
+      if: runner.os == 'Windows' || runner.os == 'macOS'
+      shell: bash
+      run: echo "PYTHON=python" >> "$GITHUB_ENV"
+    - name: Set the name for the khiops laucher script
+      shell: bash
+      run: |
+        if [[ $RUNNER_OS == "Windows" ]]
+        then
+          # Put the full path in windows since bash doesn't find ".cmd" files
+          # in the path even if they are there
+          modl_path=$(which MODL)
+          khiops_bin_dir=$(dirname "$modl_path")
+          KHIOPS_SCRIPT="$khiops_bin_dir/khiops.cmd"
+          KHIOPS_CC_SCRIPT="$khiops_bin_dir/khiops_coclustering.cmd"
+          echo "KHIOPS_SCRIPT=$KHIOPS_SCRIPT" >> "$GITHUB_ENV"
+          echo "KHIOPS_CC_SCRIPT=$KHIOPS_CC_SCRIPT" >> "$GITHUB_ENV"
+        else
+          echo "KHIOPS_SCRIPT=khiops" >> "$GITHUB_ENV"
+          echo "KHIOPS_CC_SCRIPT=khiops_coclustering" >> "$GITHUB_ENV"
+        fi
+    - name: Check Khiops installation
+      shell: bash
+      env:
+        KHIOPS_PROC_NUMBER: '4'
+        OMPI_MCA_rmaps_base_oversubscribe: 'true'
```

Comment:
The values can be left unquoted (see previous comments on this topic above).

Reply:
done